internal/pkg/cli/svc_deploy.go

```diff
@@ -309,26 +314,67 @@ func (o *deploySvcOpts) pushToECRRepo() error {
 	return o.docker.Push(uri, o.ImageTag)
 }
 
-func (o *deploySvcOpts) getDockerfilePath() (string, error) {
-	type dfPath interface {
+type dockerParams struct {
+	context    string
+	dockerfile string
+	args       map[string]string
+}
+
+// toDockerBuildArguments fills in the fields contained in the image/build tag in the manifest.
+func (d *dockerParams) toDockerBuildArguments() docker.BuildArguments {
+	return docker.BuildArguments{
+		Context:    d.context,
+		Dockerfile: d.dockerfile,
+		Args:       d.args,
+	}
+}
+
+func (o *deploySvcOpts) getDockerfile() (*dockerParams, error) {
```

Comment:
Can we ditch the `dockerParams` struct and just return `docker.BuildArguments` directly?